CONTRIBUTING.md

```diff
@@ -22,6 +23,30 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 kubectl scale deployments/inflate --replicas=3
 ```
 
+### Use local environment instead of GitHub Codespaces. 
+> Tested environment : WSL2
+
+1. **Install tools**
+   * make
+   * kubectl
+   * [helm](https://github.com/helm/helm/releases)
+   * [golang](https://go.dev/dl/) > 1.21
+   * [yq](https://github.com/mikefarah/yq/releases) / [jq](https://github.com/jqlang/jq/releases)
+   * [skaffold](https://skaffold.dev/docs/install/#standalone-binary)
+   * [azure-cli](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+2. **Provision cluster, build and deploy Karpenter** : Set `AZURE_SUBSCRIPTION_ID` to your subscription (and customize region in `Makefile-az.mk` if desired). Then run `make az-all`. This logs into Azure (follow the prompts), provisions AKS and ACR (using resource group and cluster name `$COMMON_NAME`, ACR it will not allow dashes), builds and deploys Karpenter, deploys sample `default` Provisioner and `inflate` Deployment workload.
```

Comment:
```suggestion
2. **Provision cluster, build and deploy Karpenter** : Set `AZURE_SUBSCRIPTION_ID` to your subscription (and customize region in `Makefile-az.mk` if desired). Then run `make az-all`. This logs into Azure (follow the prompts), provisions AKS and ACR (using resource group and cluster name `$COMMON_NAME`, ACR it will not allow dashes), builds and deploys Karpenter, deploys sample `general-purpose` nodepool and `inflate` deployment workload.
```

Reply:
I changed it as per your suggestion.